ckanext/datagovtheme/fanstatic_library/scripts/datagovtheme.js

```diff
@@ -1,81 +1,4 @@
-this.ckan.module('datagovtheme-site-wide-search', function($, _) {
-  return {
-    options: {
-      base_url: 'http://www.data.gov/search/node/'
-    },
-    initialize: function() {
-      var el = this.el;
-      var options = this.options;
-      $("#field-sitewide-search").change(function(){
-        el.attr('action', options.base_url + this.value);
-      });
-      this.el.submit(function(){
-        window.location.href = $(this).attr('action');
-        return false;
-      });
-    }
-  }
-});
-
-this.ckan.module('datagovtheme-search-helper-message', function($, _) {
-  return {
-    options: {},
-    initialize: function() {
-      var el = this.el;
-      var popup = $('#search-helper-message');
-      el
-        .on('mouseover', function() {
-          popup.show();
-        })
-        .on('mouseout', function() {
-          popup.hide();
-        });
-      if ($('html').hasClass('ie7')) {
-        function position() {
-          var offset = el.offset();
-          popup
-            .css('left', offset.left - popup.outerWidth() + el.outerWidth())
-            .css('top', offset.top + el.outerHeight() + 5);
-        }
-        popup.appendTo('body');
-        position();
-        $(window).on('resize', position);
-      }
-    }
-  };
-});
-
-function GetURLParameter(sParam) {
-    var sPageURL = window.location.search.substring(1);
-    var sURLVariables = sPageURL.split('&');
-    for (var i = 0; i < sURLVariables.length; i++)  {
-        var sParameterName = sURLVariables[i].split('=');
-        if (sParameterName[0] == sParam) 
-            return sParameterName[1];        		
-    }
-	
-    return 0;
-}
-
-
 $(document).ready(function () {
-
-    var sort = GetURLParameter('sort');	
-    var q = GetURLParameter('q');	
-	
-	if(sort == 0 && q == 0)
-	   $("#sort_option").html("Datasets ordered by Popular");
-	else if(sort == 'none') {
-	   $("#sort_option").html("Datasets ordered by Relevance");
-	   $("#field-order-by").val('score desc, name asc');
-	}
-	else {
-	$("#field-order-by option[value='none']").remove();
-	   var sortVal = decodeURIComponent(sort).replace(/\+/g, ' ');	   
-	   var sortTxt = $("#field-order-by option[value='" + sortVal + "']").text();
```

Comment:
Here is where `sort` parameter can be crafted to inject xss function.

Reply:
hmm....  just realized the change is already in main branch. this has been dealt with before in https://github.com/GSA/ckanext-datagovtheme/pull/90. A git rebase did the trick.